docs/users_guide/ocean/test_groups/internal_wave.rst

```diff
@@ -0,0 +1,141 @@
+.. _ocean_internal_wave:
+
+internal_wave
+=============
+
+The ``ocean/internal_wave`` test group induces internal wave propagation and is documented in 
+`Ilicak et al. (2012) <https://doi.org/10.1016/j.ocemod.2011.10.003>`_.
+
+The domain is periodic on the zonal boundaries and solid on the meridional boundaries.
+Salinity is constant throughout the domain (at 35 PSU).  The initial
+temperature has a linear background stratification with a sinusoidal perturbation in the center of the domain. This perturbation initiates symmetric waves that propage out from the center and then back to the center after meeting the solid boundaries. 
```

Comment:
```suggestion
temperature has a linear background stratification with a sinusoidal perturbation in the center 
of the domain. This perturbation initiates symmetric waves that propage out from the center and 
then back to the center after meeting the solid boundaries. 
```

Reply:
I'd like to keep the lines easier to read from the terminal, so under 80 characters or so.